src/main/java/models/Spanish.java

```diff
@@ -0,0 +1,22 @@
+package models;
+
+/**
+ * Created by Cameron on 2/18/2016.
+ */
+public class Spanish extends Deck{
+
+    public Spanish(){
+
+    }
+
+    public void buildDeck(){
+        for(int i = 1; i < 13; i++){
+            if(i==8){ i=10; }
```

Comment:
I would give the code "i=10" its own separate line here, rather than sharing a line with the condition, then you can just remove the brackets around that statement, also this part of the code will be easier to look at/understand overall.